pkg/operator/targetconfigcontroller/targetconfigcontroller.go

```diff
@@ -28,41 +34,52 @@ const (
 	cloudConfigNamespace = "openshift-config"
 )
 
+var (
+	emptyDriverConfig = opv1.CSIDriverConfigSpec{}
```

Comment:
I'm not a big fan of such global vars - IMO it does not save much `if`s below.

Reply:
fixed.